### PR TITLE
Update example.yml prefixes

### PR DESCRIPTION
**Pull request by JJRcop**
## About The Pull Request

Since @tgstation-server broke I took at look at manually writing in changelogs for the ~40+ PRs that were missed. I decided it was not something I was willing to do, but I added missing prefixes to example.yml in the process.

### CODE REVIEW

File: tools/travis/check_changelogs.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 set -euo pipefail
 
-md5sum -c - <<< "49bc6b1b9ed56c83cceb6674bd97cb34 *html/changelogs/example.yml"
+md5sum -c - <<< "364a16f858a957486eaeb1e12673c39c *html/changelogs/example.yml"
```
Comment (yoyobatty) [inline tools/travis/check_changelogs.sh]: Where exactly did you get this value? I was looking to mirror to PR.

Comment (JJRcop) [reply tools/travis/check_changelogs.sh]: md5sum

Comment (yoyobatty) [reply tools/travis/check_changelogs.sh]: How does one generate it?

Comment (JJRcop) [reply tools/travis/check_changelogs.sh]: md5sum

Comment (JJRcop) [reply tools/travis/check_changelogs.sh]: Like this: md5sum

Comment (JJRcop) [reply tools/travis/check_changelogs.sh]: md5sum

Comment (yoyobatty) [reply tools/travis/check_changelogs.sh]: oh, thank you!

**Pull request merged**